frontend/src/pages/auth/signin/SignIn.tsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
-import {Titles} from "../../../constansts/titles";
+import {Box, Button} from "@mui/material";
+import "../../../common/styles.css";
+import {useAppDispatch} from "../../../stores/redux/hooks";
+import {signIn} from "../../../stores/redux/loggedUser/loggedUserSlice";
+import {logInService} from "../../../services/user";
+import {Buttons} from "../../../constansts/buttons";
 
 const SignIn: React.FC = () => {
+    const dispatch = useAppDispatch()
+
+    const handleSignIn = async () => {
+        const loggedUser = await logInService({email: "user@mail.com", password: "password"});
+        if (loggedUser) {
```

Comment:
as we discussed, you have to make a call to user manager to get user and then create ILoggedUser

Reply:
1. send request to get user.
2. extract user id from token
3. get user by id
4. create loggedUser based on user and token